packages/react-components/src/organisms/SharedResearchOutputHeaderCard.tsx

```diff
@@ -79,7 +81,12 @@ const SharedResearchOutputHeaderCard: React.FC<
     />
     <span css={headerStyle}>
       <Display styleAsHeading={3}>{title}</Display>
-      {!published && <StateTag label="Draft" />}
+      {!published && (
+        <StateTag
+          label={reviewRequestedBy ? 'In Review' : 'Draft'}
```

Comment:
This was happening before but I've just noticed now that the tag is not in the right place on mobile view

the left side is figma and the right side how it looks now
<img width="706" alt="Screenshot 2023-06-19 at 15 41 49" src="https://github.com/yldio/asap-hub/assets/16595804/41da8aaf-13c0-4512-a094-25f67f6272ab">